src/main/schema/github-actions-typing.schema.json

```diff
@@ -0,0 +1,192 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
```

Comment:
I don't have much experience with JSON Schema, but let me ask: is there a reason to use Draft 07 instead of https://json-schema.org/draft/2020-12? The difference is almost 4 years. Is it about tools that support these versions?

Reply:
Let me quote https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices:
> Use the most recent JSON Schema version (specified by `$schema`) that's widely supported by editors and IDEs. Currently, the best supported version is `draft-07`. Later versions of JSON Schema are not recommended for use in SchemaStore until editor/IDE support improves for those versions.

So I also sticked to that, especially as I started with contributing it to schemastore until you said you want it here.
Also `draft-07` was fully sufficient for what we need.
No idea what the other drafts would bring in addition.